### DISCUSSION TITLE
Reading tag through door

### DISCUSSION THREAD

**Discussion by rocket59** [adonno/tagreader#177] [Q&A] [answered]
Looks like a great project ! 

Has anyone tested the tag reading capability of the PN532 through a timber door? It makes sense to place the reader behind the door rather than exposed outside.

**Comment by rocket59** [comment, marked as answer]
OK I'll answer my own question - card yes, tag no

**Comment by abreups** [comment]
For me it worked through a 1.5cm wood door but didn't work through a 3.5cm door.
